Assets/Scripts/Ship/EnemyShip.cs

```diff
@@ -36,6 +37,7 @@ public void SetTargets(PlayerShip[] players)
         protected override void Movement()
         {
             transform.position = transform.position + _delta * _speed;
+            GameContext.EnemysData.Positions = new[] { transform.position.x, transform.position.y, transform.position.z };
```

Comment:
We may have an unknown number of enemies.
Therefore, in order to identify them, we can use a unique identifier.
private int _id;
This identifier will be equal to the index in the array where the enemy ship is located.
It must be passed when creating a ship in the factory.
It makes sense to do the same with the players, then we can make it so that there will be as many players as we like

Reply:
i don't  know how to do this